rosetta/services/constants.go

```diff
@@ -1,6 +1,20 @@
 package services
 
+import "os"
+
+func getEnvNetwork() string {
+	if value, ok := os.LookupEnv("NETWORK"); ok {
+		if value == "mainnet" || value == "testnet" {
+			return value
```

Comment:
Maybe adding here a log output saying "unrecognized network <value> . Defaulting to mainnet" or something like that ?

Reply:
Made the modification. Should be better now.